Feat: readingTime 계산 알고리즘 작성

Pangho04 opened the pull request:
<!---- <타입>: <일감 명> 의 형식으로 제목 작성 -->

## 개요

<!---- 변경 사항 및 관련 이슈가 있다면 간단하게 작성해주세요. 어떻게보다 무엇을 왜 수정했는지 설명해주세요. -->
Resolves: #09

- [x]  본문 텍스트의 총 단어 수를 구합니다.
- [x]  사용자 읽기 시간(밀리 세컨드)과 총 단어 수를 곱해 예상 리딩 타임(밀리 세컨드)을 구합니다.
- [x]  0초 또는 30초가 되도록 반올림합니다.
- [x]  밀리 세컨드를 반환합니다.
- [x]  테스트를 작성합니다.

readingTime 계산 알고리즘 추가 및 테스트 코드를 작성했습니다.
## 코드 변경 사항

<!---- 어떻게 코드를 작성 및 수정했는지 설명해주세요. 필요한 경우 코드 라인 링크를 활용해주세요. -->
readingTime 계산 알고리즘 추가를 위해, app.service.js 파일 내에 함수를 추가했습니다.
해당 함수 테스트 코드 작성을 위해, app.controller.spec.js  파일을 생성하여 테스트 코드를 작성하였습니다.
## 기타 전달 사항

<!---- 리뷰어들에게 기능관련 외에 전달하고 싶은 사항을 작성해주세요. -->

## PR Checklist

<!---- PR이 다음 요구 사항을 충족하는지 확인하세요. -->

- [x] 주어진 Task의 요구사항을 모두 작성했습니다.
- [x] 작성한 PR을 다시 한번 더 검토하였습니다.
- [x] merge 할 브랜치의 위치를 확인하였습니다.

Pangho04 committed:
Rename: 초기 요소 삭제

- 불필요한 초기 요소 삭제

Pangho04 committed:
Add: getReadingTime Test 작성

Pangho04 committed:
Feat: readingTime 계산 함수 추가

- 아티클 본문, 사용자 WPM을 받아서 readingTime을 ms단위로 반환 합니다.

JieunZoeyOh commented on src/app.controller.spec.js:
```diff
@@ -0,0 +1,34 @@
+      expect(appService.getReadingTime(articleBody, userWPM)).toBe(90000);
+    });
+
+    it("should return 90,000ms", () => {
```
```suggestion
    it("should return 60,000ms", () => {
```

60,000ms이 더 적합해보입니다.

JieunZoeyOh reviewed (changes requested):
고생하셨습니다. 리뷰 확인 부탁드릴게요!

dlgnswk commented on src/app.service.js:
```diff
@@ -5,4 +5,25 @@ export class AppService {
+  getReadingTime(articleBody, userWPM) {
+    const numOfWords = articleBody.split(" ").length;
+    const rawReadingTime = (numOfWords / userWPM) * 60 * 1000;
+    let readingMinute = Math.floor(rawReadingTime / 1000 / 60);
```
readingSeconds 와 같이 readingMinutes 로 유사한 형태의 변수명을 사용하는것은 어떨까요?
```suggestion
    let readingMinutes = Math.floor(rawReadingTime / 1000 / 60);
```

dlgnswk reviewed (changes requested):
코멘트 확인 후 수정 부탁드립니다!

Pangho04 committed:
Style: 오탈자 수정

- 테스트 Unit 설명문 내 '90,000ms -> 60,000ms'.
- 변수명 단위 일치: readingMinute -> readingMinutes.
